src/main/java/org/folio/rest/impl/OaiPmhViewInstancesAPI.java

```diff
@@ -80,6 +77,35 @@ public void getOaiPmhViewInstances(String startDate, String endDate, boolean del
     }
   }
 
+  @Override
+  public void getOaiPmhViewUpdatedInstanceIds(String
+                                                startDate, String endDate, boolean deletedRecordSupport,
+                                              boolean skipSuppressedFromDiscoveryRecords, @Pattern(regexp = "[a-zA-Z]{2}") String lang, RoutingContext routingContext,
+                                              Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
```

Comment:
Please use the standard line continuation and 2 space indentation

Reply:
Done